src/averageAges.js

```diff
@@ -14,12 +14,16 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = 0) {
+  const mailPeople = people.filter(person => {
+    return person.sex === 'm'
+      && (!century || Math.ceil(person.died / 100) === century);
+  });
+
+  const sumOfAges = mailPeople.reduce((total, person) =>
+    total + person.died - person.born, 0);
```

Comment:
Не розривай колбеки, так важко зрозуміти, де він закінчується
```suggestion
  const sumOfAges = mailPeople.reduce(
    (total, person) => total + person.died - person.born
  , 0);
```